TESTING.md

```diff
@@ -94,3 +94,18 @@ datastream_id: derived:com.google.step_count.delta:com.google.android.gms:LGE:LG
 
 We will be using the dedicated LG Watch datasource for the moment.
 
+## Testing APIs
+Check - with cURL, from command line - the following API endpoints to be up and running.
```

Comment:
@cretualexandru: I think it would be useful to also have at least a prebuilt cURL command in the doc, so it will leave less room for interpretation.